wallet-new/test/unit/Test/Spec/Translation.hs

```diff
@@ -343,11 +340,8 @@ data Disagreement h a =
 
     -- | Both Cardano and the DSL reported the chain as valid, but they computed
     -- a different UTxO
-  | UnexpectedUtxo {
-        utxoDsl     :: Utxo h a
-      , utxoCardano :: Cardano.Utxo
-      , utxoInt     :: Cardano.Utxo
-      }
+    -- UnexpectedUtxo utxoDsl utxoCardano utxoInt
+  | UnexpectedUtxo !(Utxo h a) !Cardano.Utxo !Cardano.Utxo
```

Comment:
This part here seems a little confusing to me. I feel like we should have `newtype`s for each these fields so things become a little more clear. What do you think @mhuesch?

Reply:
@mhuesch perhaps we just leave the comment above then? If we're not going to use `newtype`s then I think the comment could be helpful.

Reply:
@intricate oh, yeah definitely! I forgot that would be deleted and missed your point. Especially since there are two identical types, we should definitely document why they're different, and which field does what.

Something similar to [the section below](https://github.com/input-output-hk/cardano-sl/pull/3251/files/b5559bf60cdf086af4d2d0c0210028f53ceb093c#diff-edb9c32ecf513a45a45c9deb22224750R268) seems good.